src/mlpack/tests/main_tests/logistic_regression_test.cpp

```diff
@@ -627,3 +627,25 @@ TEST_CASE_METHOD(LogisticRegressionTestFixture, "LRDecisionBoundaryTest",
   // Check that the output changed when the decision boundary moved.
   REQUIRE(arma::accu(output1 != output2) > 0);
 }
+
+/**
+  * Check that running the binding with print_training_accuracy set to true
+  * does not crash.
```

Comment:
```suggestion
 * Check that running the binding with print_training_accuracy set to true
 * does not crash.
```
(Really trivial spacing fix)